src/build.cc

```diff
@@ -614,6 +645,10 @@ vector<Edge*> RealCommandRunner::GetActiveEdges() {
 
 void RealCommandRunner::Abort() {
   subprocs_.Clear();
+// TODO: jobserver client support for Windows
+#ifndef _WIN32
+  jobserver_->Clear();
+#endif
```

Comment:
Will this be called after SIGTERM/SIGINT? The specification recommends so.

Reply:
yes, that's exactly the reason I put it there.